libraries/AP_HAL_SITL/sitl_gps.cpp

```diff
@@ -756,7 +756,7 @@ void SITL_State::_update_gps_nmea(const struct gps_data *d, uint8_t instance)
                      dstring);
 
     if (_sitl->gps_hdg_enabled[instance]) {
-        _gps_nmea_printf(instance, "$GPHDT,%.2f,T", d->yaw);
+        _gps_nmea_printf(instance, "$GPTHS,%.2f,%c,T", d->yaw, d->have_lock ? 'A' : 'V');
```

Comment:
let's make it so that SIM_GPS_HDG==1 means HDT and SIM_GPS_HDG==2 means THS, so we can test both